secboot/secboot_tpm.go

```diff
@@ -99,12 +99,26 @@ func CheckTPMKeySealingSupported() error {
 		logger.Noticef("TPM device detected but not enabled")
 		return fmt.Errorf("TPM device is not enabled")
 	}
+	if lockoutAuthSet(tpm) {
```

Comment:
This is too simplistic as the factory reset code is also using the "CheckTPMKeySealingSupported" check and it can reset the TPM so this needs to be done in a more targeted way for the installer API only.

Reply:
This is now resolved